src/docs/integrations/slack/index.mdx

```diff
@@ -75,16 +88,23 @@ You’ll see “Verified” when you’ve entered the correct URL.
 
 _Note: When you enter the Request URL, Slack tries to verify it. The request will fail if you didn’t configure the client keys/secret and restarted Sentry to ensure those config changes were picked up. If it fails to validate, first make sure your Sentry instance is running and Slack can talk to it (Slack [doesn’t have an ip range](https://twitter.com/slackapi/status/567110311476350976?lang=en))_
 
-On the same page under __Selected Events__, click __Add Workspace Event__ and search for `link_shared`:
+Still on the __Event Subscription__ page, go to __Subscribe to bot events__ and add the following bot user events:
 
-On the same page under __App Unfurl Domains__, click __Add Domain__ to enter your domain and click Done.
+* `link_shared`
+* `message.im`
 
-At the bottom of the page, click __Save Changes__.
+and next go to __Subscribe to events on behalf of users__ and add the following event:
+
+* `link_shared`
 
-Navigate to __Bot Users__
+Lastly, still on that page, go to __App Unfurl Domains__, click __Add Domain__ to enter your domain and click Done.
 
-Click __Add a Bot User__ and give your sentry app bot a name (this is the name that will be displayed when alerts are triggered)
+At the bottom of the page, click __Save Changes__.
+
+Navigate to __App Home__, the first section under __Features__. Here is where you can edit your bot display name (this is the name that will be displayed when alerts are triggered)
 
 ![Bot](./bot.png)
 
-Now you can use Slack with Sentry! See our [documentation on configuring the Slack integration for your projects](https://docs.sentry.io/workflow/integrations/global-integrations/#slack) to take advantage of multi-channel Alert routing with the new integration.
+If you plan on having Slack alerts sent to specific users (`@person`) in addition to channels, toggle on the __Messages Tab__ under __Show Tabs__ in the __App Home__ page.
```

Comment:
>toggle on the __Messages Tab__ under __Show Tabs__ in the __App Home__ page.

toggle the __Messages Tab__ under __Show Tabs__ on inside the __App Home__ page.